src/amo/reducers/collections.js

```diff
@@ -680,18 +680,23 @@ export const updateCollectionAddon = ({
   };
 };
 
-// The DeleteCollectionAddonNotesAction is identical to the
-// UpdateCollectionAddonAction, with the only difference being that
-// we pass '' into the notes field. Therefore we can reuse the
-// UpdateCollectionAddonParams type to describe the payload.
+type DeleteCollectionAddonNotesParams = {|
+  addonId: number,
+  errorHandlerId: string,
+  notes?: string,
```

Comment:
mmh I guess you added this because you need it for the action's payload. The problem is that now, the action's payload could not have `notes`. Flow would not say anything and I have the feeling that having `notes` would be useful to delete a note.

Maybe we should keep `UpdateCollectionAddonParams` for the parameters and have `DeleteCollectionAddonNotesParams` with `notes` required.

Reply:
I'm not sure I fully understand what you've suggested, but what I did was remove `notes` from `DeleteCollectionAddonNotesParams` and used `UpdateCollectionAddonParams` to describe the payload, and that seems to work and I think it supports your case.